### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by Carlos-Tirado**
hey guys, some styling changes on filter game options

### CODE REVIEW

File: src/components/HeaderMenu.js [hunk 1]
```diff
@@ -28,8 +28,8 @@ function Header(token, setToken) {
         axios
             .post(
                 `https://teammate-app.herokuapp.com/auth/token/logout/`,
-                {},
-                {headers: { Authorization:`Token$ {token}`}}
+                {}, {withCredentials: true},
```
Comment (racheldk) [inline src/components/HeaderMenu.js:31]: What does line 31 do here?

File: src/pages/myGamesPage.js [hunk 2]
```diff
@@ -61,10 +62,10 @@ export default function MyGames({
 
     return (
         <>
-            <div className="app-body">
+            <div className="app-body" style={{textAlign:"center"}}>
```
Comment (racheldk) [inline src/pages/myGamesPage.js:65]: I haven't seen a ton of inline styling so far in this project, does using some inline styling affect how the CSS classes work?

File: src/pages/myGamesPage.js [hunk 3]
```diff
@@ -61,10 +62,10 @@ export default function MyGames({
...
             <Header />
             <div className="nav_links">
-                <button className="buttonmygames" onClick={() => handleMenuConfirmed()}>
+                <button className="buttonmtgames" onClick={() => handleMenuConfirmed()}>
```
Comment (racheldk) [inline src/pages/myGamesPage.js:68]: Did you mean for this to be buttonmygames?

File: src/components/HeaderMenu.js (hunk 1 above)
Comment (Carlos-Tirado) [reply src/components/HeaderMenu.js:31]: works like the token authorization, I was just trying something.

File: src/pages/myGamesPage.js (hunk 3 above)
Comment (Carlos-Tirado) [reply src/pages/myGamesPage.js:68]: oh god, misspelled issue, sorry about it.

File: src/pages/myGamesPage.js (hunk 2 above)
Comment (Carlos-Tirado) [reply src/pages/myGamesPage.js:65]: both works, but I'm not using that text align prop in css anymore. I used it there days ago before to start working from App.css

File: src/components/HeaderMenu.js (hunk 1 above)
Comment (racheldk) [reply src/components/HeaderMenu.js:31]: Cool! I haven't seen that before. Do we need to change anything about login if we add this here?

File: src/pages/myGamesPage.js (hunk 3 above)
Comment (racheldk) [reply src/pages/myGamesPage.js:68]: Not a problem!

File: src/pages/myGamesPage.js (hunk 2 above)
Comment (samrespass) [reply src/pages/myGamesPage.js:65]: Yea we'll want to keep the styling contained to either the app.css file or the chakra tags for readability and ease of editing

File: src/pages/myGamesPage.js (hunk 3 above)
Comment (samrespass) [reply src/pages/myGamesPage.js:68]: For class names let's stick to the word-word format

File: src/components/gamesList.js [hunk 4]
```diff
@@ -194,75 +194,75 @@ export default function GamesList({ token, games, listType, listTitle }) {
...
-                                        </button-delete>
-                                        {/* <button onClick={()=> handleEditMyGame(game) }>Edit</button> */}
-                                        <Link to={`/edit/${game.id}`} token={token} game={game}>
-                                            Edit
-                                        </Link>
-                                    </>
-                                );
-                            case "confirmed":
-                                return (
-                                    <button-to-cancel
```
Comment (samrespass) [inline src/components/gamesList.js:254]: Are these's being used as button classes? We'll actually want to use the chakra <Button> component and style  it within there

File: src/App.css [hunk 5]
```diff
@@ -185,7 +180,28 @@ body {
     padding: 0 1em;
     width: 100%;
 }
+h1{ color:#285E61; text-align: center; font-family: Verdana;
```
Comment (samrespass) [inline src/App.css:183]: Is this style for working with the chakra <Header> component or  h1 tags somewhere in the code?

File: src/pages/myGamesPage.js (hunk 3 above)
Comment (Carlos-Tirado) [reply src/pages/myGamesPage.js:68]: ok, use dash, got it

File: src/App.css (hunk 5 above)
Comment (Carlos-Tirado) [reply src/App.css:183]: that belongs to the "Open Games List"  tittle in OpenGamesPage.js  page

File: src/components/HeaderMenu.js (hunk 1 above)
Comment (Carlos-Tirado) [reply src/components/HeaderMenu.js:31]: no we don't have to, If i remove it it still works just with {headers: { Authorization:`Token$ {token}`}}

File: src/components/gamesList.js (hunk 4 above)
Comment (Carlos-Tirado) [reply src/components/gamesList.js:254]: If we a chance today, let's talk (zoom) about those details.

**Pull request merged**